<launch>
    <arg name="sensor1" default="tactip"/>
    <arg name="sensor2" default="uskin"/>
    <arg name="checkpoint1" default="DI_tactip"/>
    <arg name="checkpoint2" default="tactip_uskin"/>
    <arg name="cam_id" default="10"/>

    <node pkg="genforce" type="tactip_publisher.py" name="tactip_publisher" output="screen">
        <param name="cam_id" value="$(arg cam_id)"/>
        <param name="sensor_id" value="$(arg sensor1)"/>
    </node>

    <node pkg="genforce" type="uskin_publisher.py" name="uskin_publisher" output="screen" />

    <node pkg="genforce" type="force_slip.py" name="force_tactip" output="screen" >
            <param name="sensor_id" value="$(arg sensor1)"/>
            <param name="checkpoint" value="$(arg checkpoint1)"/>
    </node>

    <node pkg="genforce" type="force_slip.py" name="force_uskin" output="screen" >
            <param name="sensor_id" value="$(arg sensor2)"/>
            <param name="checkpoint" value="$(arg checkpoint2)"/>
    </node>

</launch>

